pyquil/operator_estimation.py

```diff
@@ -37,8 +32,13 @@
     to_json,
     zeros_state,
 )
+from pyquil.experiment._group import (
+    _max_weight_state,
+    _max_weight_operator,
+    group_settings as group_experiments,
+)
```

Comment:
there are definitely some missing here -- `construct_tpb_graph`, `group_experiments_clique_removal`, and `group_experiments_greedy`